rpc/blockchain.go

```diff
@@ -28,16 +30,31 @@ import (
 type PublicBlockchainService struct {
 	hmy     *hmy.Harmony
 	version Version
+	limiter *rate.Limiter
 }
 
+var (
+	DefaultRateLimiterWaitTimeout = 5 * time.Second
+)
+
 // NewPublicBlockchainAPI creates a new API for the RPC interface
-func NewPublicBlockchainAPI(hmy *hmy.Harmony, version Version) rpc.API {
-	return rpc.API{
-		Namespace: version.Namespace(),
-		Version:   APIVersion,
-		Service:   &PublicBlockchainService{hmy, version},
-		Public:    true,
+func NewPublicBlockchainAPI(hmy *hmy.Harmony, version Version, limiterEnable bool, limit int) rpc.API {
+	if limiterEnable {
```

Comment:
What is the case for not limiting the rate?

Reply:
Now for the explorer node for the time being there is no unrestricted scenario, but I think this switch is still exposed as much as possible, this is a functional switch